### PR TITLE
Add Spinner component for CallbackPage

### PR DESCRIPTION
**Pull request by RyoYamashita765**
スクリーンショット 2021-11-11 12 07 04

### CODE REVIEW

File: cloud_dashboard/src/organisms/CallbackForm.tsx [hunk 1]
```diff
@@ -51,8 +52,19 @@ const CallbackForm = () => {
...
+      <Loader
+        type="Grid"
+        color="#00BFFF"
+        height="3rem"
+        width="3rem"
+      />
+      <span style={{
+        fontSize: '2rem',
+        marginLeft: '1rem'
+      }}>Please waiting...</span>
```
Comment (naoi) [inline cloud_dashboard/src/organisms/CallbackForm.tsx:65]: @RyoYamashita765 It is natural to say:

```suggestion
      }}>One moment, please...</span>
```

**Review by naoi — CHANGES_REQUESTED** [review]

File: cloud_dashboard/src/organisms/CallbackForm.tsx (hunk 1 above)
Comment (RyoYamashita765) [reply cloud_dashboard/src/organisms/CallbackForm.tsx]: @naoi I fixed this message. Please check it.

**Comment by naoi** [comment]
Thank you for the update.  Could you please fix the following error while applying the patch?

```
$ git apply -v /tmp/33.diff.1 
Checking patch LICENSE...
/tmp/33.diff.1:280: new blank line at EOF.
+
Checking patch cloud_dashboard/package.json...
Checking patch cloud_dashboard/src/App.tsx...
Checking patch cloud_dashboard/src/organisms/CallbackForm.tsx...
Checking patch cloud_dashboard/yarn.lock...
Checking patch css/2.chunk.css...
Checking patch js/2.chunk.js...
Checking patch js/3.chunk.js...
Checking patch js/main.chunk.js...
Checking patch js/runtime-main.js...
Applied patch LICENSE cleanly.
Applied patch cloud_dashboard/package.json cleanly.
Applied patch cloud_dashboard/src/App.tsx cleanly.
Applied patch cloud_dashboard/src/organisms/CallbackForm.tsx cleanly.
Applied patch cloud_dashboard/yarn.lock cleanly.
Applied patch css/2.chunk.css cleanly.
Applied patch js/2.chunk.js cleanly.
Applied patch js/3.chunk.js cleanly.
Applied patch js/main.chunk.js cleanly.
Applied patch js/runtime-main.js cleanly.
warning: 1 line adds whitespace errors.
```

Thanks!

**Comment by RyoYamashita765** [comment]
> warning: 1 line adds whitespace errors.
I do not know why you are getting this Warning.
But am I correct in understanding that the patch itself can be applied and does not interfere with the operation?

**Comment by naoi** [comment]
@RyoYamashita765 The patch itself is correct.  On the other hand, can you remove the CRLF (a blank line) at the end of line of `LICENSE`?  I'm using `git` command on Ubuntu 20.04.3 LTS.

**Comment by RyoYamashita765** [comment]
@naoi I understand. I removed CRLF at the end of the LICENCE file.

File: cloud_dashboard/src/organisms/CallbackForm.tsx [hunk 2]
```diff
@@ -51,8 +52,19 @@ const CallbackForm = () => {
...
+      <Loader
+        type="Grid"
+        color="#00BFFF"
+        height="3rem"
+        width="3rem"
+      />
+      <span style={{
+        fontSize: '2rem',
+        marginLeft: '1rem'
+      }}>One moment, please...</span>
```
Comment (naoi) [inline cloud_dashboard/src/organisms/CallbackForm.tsx:65]: @RyoYamashita765 What about this?

```suggestion
        marginLeft: '2rem'
      }}>Orchestrating the clouds...</span>
```

**Review by naoi — CHANGES_REQUESTED** [review]
@RyoYamashita765 Thank you for the update.  Now the patch is ready but I would like you to review my suggestion.  Thanks!

**Comment by RyoYamashita765** [comment]
@naoi I changed the message when the login process is being performed. Please check it.

スクリーンショット 2021-11-12 14 00 45

File: cloud_dashboard/src/organisms/CallbackForm.tsx [hunk 3]
```diff
@@ -51,8 +52,19 @@ const CallbackForm = () => {
...
+      <Loader
+        type="Grid"
+        color="#00BFFF"
+        height="3rem"
+        width="3rem"
+      />
+      <span style={{
+        fontSize: '2rem',
+        marginLeft: '2rem'
+      }}>Orchestrating the clouds...</span>
```
Comment (naoi) [inline cloud_dashboard/src/organisms/CallbackForm.tsx:65]: @RyoYamashita765 Thank you for the update.  On my second thought, can you change to:

```suggestion
      }}>Orchestrating clouds...</span>
```

**Review by naoi — CHANGES_REQUESTED** [review]
@RyoYamashita765 Thank you for the update.  I posted my comment.  Also, can you change the label of the login button from `login` to `Log into Cloud Dashboard`?  Thanks!

File: cloud_dashboard/src/organisms/CallbackForm.tsx (hunk 3 above)
Comment (RyoYamashita765) [reply cloud_dashboard/src/organisms/CallbackForm.tsx:65]: @naoi I changed the button and the message. Is there any problem with this design?

スクリーンショット 2021-11-12 15 31 07

スクリーンショット 2021-11-12 15 31 15

**Comment by naoi** [comment]
@RyoYamashita765  As I mentioned in my previous comment, can you change the label of the login button from `login` to `Log in`? Thanks!

**Comment by RyoYamashita765** [comment]
@naoi I changed the button text from "Log into Clouddashboard" to "Log in". Is this correct?
スクリーンショット 2021-11-12 15 43 57

**Comment by naoi** [comment]
@RyoYamashita765 Yes, that is correct.  Can you push the code, please?

**Comment by naoi** [comment]
@RyoYamashita765 Thank you for the update.  Perfect, I'll merge this PR.

**Pull request merged**